src/screens/About/About.style.tsx

```diff
@@ -23,3 +23,34 @@ export const Header = styled.div`
 export const TextContent = styled.div`
   max-width: 600px;
 `;
+
+export const ActiveAlumniButtonContainer = styled.div`
+  display: flex;
+  width: 150px;
+  border: 1px solid black;
+  box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.2);
+  margin-bottom: 2rem;
+`;
+
+export const ActiveAlumniButton = styled.div<{
+  teamList?: string;
+}>`
+  width: 50%;
+  display: flex;
+  justify-content: center;
+  cursor: pointer;
+  font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
+  padding: 0.1rem;
+
+  &:first-child {
+    border-right: 1px solid black;
+    background-color: ${({ teamList }) =>
+      teamList === 'active' && 'rgba(59,188,230,.5)'};
```

Comment:
nit (feel free to ignore for now): Using magic string values like this is a bit fragile (you could typo the string somewhere by accident) and not self-documenting (there's no easy way to tell what the valid values are).  

It'd be a bit cleaner to use an enum or a string union.

**Enum**
```
enum TeamList {
  Active,
  Alumni,
}
```
Then change `'active'` here to `TeamList.Active`

**String Union**

```
type TeamList = 'active' | 'alumni';
```

and then change the property type above from `teamList?: string` to `teamList?: TeamList`.  And now TypeScript will yell at you if you pass any string other than 'active' or 'alumni'.

Reply:
ooo yes. enum it is. thank you!